site/docs/concepts/web-app.md

```diff
@@ -0,0 +1,272 @@
+
+4. Use the **Specification Editor** to edit collection specifications, if desired. If you make any changes, click **Next** again.
+
+5. Click **Save and Publish**.
+
+Editing a capture only affects how it will work going forward.
+Data that was captured before editing will reflect the original configuration.
+
+It's important to be mindful of how your edits will effect downstream processes. For more information on the implications of editing, see the [reference documentation](../reference/editing.md).
+
+### Creating a capture
+
+When you click **Create Capture**, you're taken to the Create Capture page.
+In the first view, all available capture connectors are displayed.
+
+Select the tile of the system from which you want to capture data to show the full capture form.
+The form details are specific to the connector you chose.
+
+For detailed steps to create a capture, see the [guide](../guides/create-dataflow.md#create-a-capture).
+
+After you successfully publish a capture, you're given the option to materialize the collections you just captured.
+You can proceed to the materialization, or opt to exit to a different page of the web app.
+
+## Collections page
+
+The **Collections** page shows a read-only table of [collections](./collections.md) to which you have access.
+You can view each collection's specification and see a sample of its data.
+This can help you verify that collection data was captured as expected and that you'll be able to materialize it how you want, and troubleshoot it necessary.
+
+To reveal the **Specification** and **Data Preview** sections, expand **Details** next to a collection name.
+
+The **Specification** section shows the collection specification as JSON in a read-only editor.
+(If you need to modify a collection, edit the [capture](#editing-captures) that it came from.)
+
+![](<./webapp-images/collections-view-schema.png>)
+
+The **Data Preview** section shows a sample of collection [documents](./collections.md#documents): the individual JSON files that comprise the collection.
+Documents are organized by their collection key value. Click a key from the list to view its document.
+
+![](<./webapp-images/collections-view-data.png>)
+
+## Materializations page
+
+The **Materializations** page shows you a table of existing Flow [materializations](./materialization.md) to which you have [access](../reference/authentication.md).
+The **New Materialization** button is also visible.
+
+You use the table to monitor your materializations. It's nearly identical to the table on the [Captures page](#captures-page), with a few exceptions.
+
+![](<./webapp-images/materialization-page.png>)
+
+**1:** Select all or deselect all.
+
+**2:** Enable, Disable, and Delete buttons. These actions will be applied to the selected table rows. Choose **Disable** to temporarily pause the flow of data, **Enable** to resume, and **Delete** to permanently remove the materialization(s).
+
+**3:** Filter materializations by name.
```

Comment:
See the comment left behind in the _Captures page_ section regarding the search bar.

Reply:
Doing this edit really highlighted the difference in naming convention between captures and materializations -- namely that captures have the connector name appended to them, and materializations do not. Search seems like a good reason to have the connector name included for both entities.

Reply:
Well, right now you can enter a materialization connector name directly in the search bar and it will filter by connector accordingly. The search bar is essentially looking for query matches in the catalog name **and** connector name of the data populating a given task table. For example, if you type _google sheets_ into the search bar of the materialization table, you will see that many of the results are connector name matches (i.e., the catalog name of the match does not contain the string _google sheets_.